spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
+  it 'should display the names of dishes' do
```

Comment:
check the user story on this one and consider including the names of dishes + their prices here 👍